pkg/connectors/openapiclient/README.md

```diff
@@ -0,0 +1,112 @@
+# Go API client for openapiclient
```

Comment:
not sure if this README.md file is required. @roee88 can you please give your opinion on this? Thanks.

Reply:
Done